Header header

string[] name
float64[] pos
float64[] vel
float64[] tau_est
float64[] pos_des
float64[] vel_des

float64 kp
float64 kd